executor/src/main/java/io/kestra/executor/ExecutorService.java

```diff
@@ -283,6 +298,10 @@ private Optional<WorkerTaskResult> childWorkerTaskResult(FlowWithSource flow, Ex
                     return Optional.of(new WorkerTaskResult(workerTaskResult
                         .getTaskRun()
                         .withOutputs(variables)
+                        .withAssets(new AssetsInOut(
```

Comment:
I don't think any flowable task would generate assets.
It may be better to restrict assets to worker task, you can do that by updating the existing validation code that prevent some fields to be used outside runnable tasks.
At some point RunnableTask and FlowableTask would be refactoted to not share anymore such attributes that cannot be used in both